src/main/java/dev/enjarai/trickster/item/EvaluationMirrorItem.java

```diff
@@ -26,6 +29,9 @@ public TypedActionResult<ItemStack> use(World world, PlayerEntity user, Hand han
         var stack = user.getStackInHand(hand);
         var otherStack = user.getStackInHand(hand == Hand.MAIN_HAND ? Hand.OFF_HAND : Hand.MAIN_HAND);
 
+        ItemStack ring = SlotReference.of(user, "ring", 0).getStack();
+        var mapComponent = MacroComponent.getMap(ring);
```

Comment:
Players can wear multiple rings, so maps from those should probably be merged if they are.

Reply:
Agreed.

Reply:
I've dealt with this in https://github.com/enjarai/trickster/pull/62/commits/5d46bb7b2a2775b1b0a44bcd48e297ec391e9a11, it should merge the first and second slot (I don't think you can have more than two rings?).